### PR TITLE
Fixed #408 : Switch `rem` & `em` `line-height` instances to `px`

### LINKED ISSUE #408: Switch `rem` & `em` `line-height` instances to `px`
There are 3 instances where `px` are not used for `line-height` in `style.css`, this is a violation of the WordPress CSS Coding Standards

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#values:

> "Line height should also be unit-less, unless necessary to be defined as a specific pixel value. This is more than just a style convention, but is worth mentioning here."
> More information: http://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/

----

• `2264:15  ✖  Unexpected unit "rem" for declaration-property-unit-whitelist property "line-height"`

https://github.com/WordPress/twentytwenty/blob/master/style.css#L2264

```css
.modal-search-form .search-field::-moz-placeholder {
	color: inherit;
	line-height: 8rem;
}
```

• `3008:15  ✖  Unexpected unit "em" for declaration-property-unit-whitelist property "line-height"`

https://github.com/WordPress/twentytwenty/blob/master/style.css#L3008

```css
.wp-block-pullquote::before {
	background: #fff;
	border-radius: 50%;
	color: #cd2653;
	content: "”";
	display: block;
	font-size: 6.2rem;
	font-weight: 500;
	line-height: 1.2em;
	margin: 0 auto 1.5rem auto;
	text-align: center;
	height: 4.4rem;
	width: 4.4rem;
}
```

• `4499:16  ✖  Unexpected unit "rem" for declaration-property-unit-whitelist property "line-height"`

https://github.com/WordPress/twentytwenty/blob/master/style.css#L4499

```css
	.modal-search-form .search-field::-moz-placeholder {
		line-height: 18.3rem;
	}
```

----

The `line-height` should be changed to the correct `px` value for each.

### PR DESCRIPTION
**Pull request by tm-const**
The line-height changed to the correct px value for each.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -3005,7 +3005,7 @@ ul.wp-block-gallery:not(.alignwide):not(.alignfull):not(.alignright):not(.alignl
 	display: block;
 	font-size: 6.2rem;
 	font-weight: 500;
-	line-height: 1.2em;
+	line-height: 12px;
```
Comment (andersnoren) [inline style.css]: This should be changed to `line-height: 1.2`, to make sure the relative line height is retained when the font size is increased.

File: style.css [hunk 2]
```diff
@@ -2261,7 +2261,7 @@ button.search-untoggle {
 
 .modal-search-form .search-field::-moz-placeholder {
 	color: inherit;
-	line-height: 8rem;
+	line-height: 128px;
```
Comment (andersnoren) [inline style.css]: 1rem equals 10px in Twenty Twenty, so you'll need to update this one accordingly.

File: style.css [hunk 3]
```diff
@@ -4496,7 +4496,7 @@ a.to-the-top > * {
 	}
 
 	.modal-search-form .search-field::-moz-placeholder {
-		line-height: 18.3rem;
+		line-height: 292.8px;
```
Comment (andersnoren) [inline style.css]: Same as for the first one (1rem = 10px).

**Comment by carolinan** [comment]
This has been fixed with https://github.com/WordPress/twentytwenty/pull/427,
closing.


@tm-const what is your wordpress.org username?

**Comment by tm-const** [comment]
Wordpress.org username: tmconst

**Pull request closed without merging**